README.md

```diff
@@ -2,6 +2,8 @@
 [![Build Status](https://travis-ci.org/graphql-devise/graphql_devise.svg?branch=master)](https://travis-ci.org/graphql-devise/graphql_devise)
 [![Gem Version](https://badge.fury.io/rb/graphql_devise.svg)](https://badge.fury.io/rb/graphql_devise)
 
+GraphqlDevise combines the power of Devise authentication with the simplicity of GraphQL API calls using JWT through DeviseTokenAuth.
```

Comment:
I see now that we were missing this line but is actually located down on line 21. I think we would like to keep it a bit simpler here on this line (5) but with the original text please
```
GraphQL interface on top of the [Devise Token Auth](https://github.com/lynndylanhurley/devise_token_auth) (DTA) gem.
```

Reply:
Ok, I'll replace my text with yours by moving it up to the top.